core/src/main/scala/chisel3/internal/BiConnect.scala

```diff
@@ -120,15 +122,24 @@ private[chisel3] object BiConnect {
         val notStrict =
           Seq(left_r.compileOptions, right_r.compileOptions).contains(ExplicitCompileOptions.NotStrict)
         if (notStrict) {
-          // chisel3 <> is commutative but FIRRTL <- is not
-          val flipped = {
-            import ActualDirection._
-            // Everything is flipped when it's the port of a child
-            val childPort = left_r._parent.get != context_mod
-            val isFlipped = Seq(Bidirectional(Flipped), Input).contains(left_r.direction)
-            isFlipped ^ childPort
+          // Traces flow from a child Data to its parent
+          @tailrec def traceFlow(acc: Boolean, data: Data): Boolean = {
```

Comment:
what is `acc` short for? maybe use a more descriptive word?

Reply:
I use it for `accumulator` although this isn't really an accumulation, more of an "accumulated result", any thoughts for a better name?

Reply:
You could consider `accumulatedFlow`. [A similar method in CIRCT uses that name](https://github.com/llvm/circt/blob/7e06c705c1ad1fff384fefe740ddae3ff2bf07f1/lib/Dialect/FIRRTL/FIRRTLOps.cpp#L98) after receiving similar feedback that `acc` was unclear.

Reply:
Done.